.travis.yml

```diff
@@ -1,11 +1,126 @@
+# 'sudo required' will give us 7,5 GB memory, but has slower startup times, as we start a VM instead of a Container
+sudo: required
 language: python
+cache: pip
+
+python: 2.7
+
+jobs:
+  include:
+    - stage: lint
+      addons: {}
+      before_install: skip
+      install:
+        - pip install flake8 flake8-import-order planemo
+        - planemo --version
+        - git diff --quiet "$TRAVIS_COMMIT_RANGE" -- ; GIT_DIFF_EXIT_CODE=$?
```

Comment:
you don't really need all of this below, as you only have one single tool here ... so we can remove this 
On the other hand I do not think it hurts ...